docs/activities.md

```diff
@@ -1,81 +1,69 @@
 
-## Activity Task Routing through Task Queues
+### Activity Id
 
-Activities are dispatched to workers through task queues. Task queues are queues that workers listen on. Task queues are highly dynamic and lightweight. They don't need to be explicitly registered. And it is okay to have one task queue per worker process. It is normal to have more than one activity type to be invoked through a single task queue. And it is normal in some cases (like host routing) to invoke the same activity type on multiple task queues.
+Use this to give your Activity a name or unique identifier that correlates to your business logic. This is optional and is not used by the Temporal server.
 
-Here are some use cases for employing multiple activity task queues in a single workflow:
+### Retry policy 
 
-- _Flow control_. A worker that consumes from a task queue asks for an activity task only when it has available capacity. So workers are never overloaded by request spikes. If activity executions are requested faster than workers can process them, they are backlogged in the task queue.
-- _Throttling_. Each activity worker can specify the maximum rate it is allowed to processes activities on a task queue. It does not exceed this limit even if it has spare capacity. There is also support for global task queue rate limiting. This limit works across all workers for the given task queue. It is frequently used to limit load on a downstream service that an activity calls into.
-- _Deploying a set of activities independently_. Think about a service that hosts activities and can be deployed independently from other activities and workflows. To send activity tasks to this service, a separate task queue is needed.
-- _Workers with different capabilities_. For example, workers on GPU boxes vs non GPU boxes. Having two separate task queues in this case allows workflows to pick which one to send activity an execution request to.
-- _Routing activity to a specific host_. For example, in the media encoding case the transform and upload activity have to run on the same host as the download one.
-- _Routing activity to a specific process_. For example, some activities load large data sets and caches it in the process. The activities that rely on this data set should be routed to the same process.
-- _Multiple priorities_. One task queue per priority and having a worker pool per priority.
-- _Versioning_. A new backwards incompatible implementation of an activity might use a different task queue.
+Unlike Workflows, Temporal does not recover the state of an Activity. Activity failures are expected to occur and when they do the Activity can be automatically retried per its Retry policy. For practical implementation guidance, read [How to retry Activities and Workflows](retry-activities-and-workflows).
```

Comment:
"Activity failures are expected to occur " - 100% true but phrase it more positively please. Not trying to hide the truth but it sort of seems like it's due to Temporal with current structure